deployments/azure-kubernetes-service-helm.yml

```diff
@@ -0,0 +1,124 @@
+# This workflow will build and push an application to a Azure Kubernetes Service (AKS) cluster when you push your code
+#
+# This workflow assumes you have already created the target AKS cluster and have created an Azure Container Registry (ACR)
+# For instructions see:
+#   - https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal
+#   - https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-portal
+#   - https://github.com/Azure/aks-create-action
```

Comment:
Too many links. Possible to have just one?

Reply:
Each of these links outline distinct and relevant concepts.  If these are to be AKS starter workflows, are we to assume they should know one or another?  In this case, I think it saves time more than it clutters the heading.